src/ansys/mechanical/core/run.py

```diff
@@ -245,3 +245,118 @@ def cli(
 
     if private_appdata:
         profile.cleanup()
+
+
+@click.command()
+@click.help_option("--help", "-h")
+@click.option(
+    "-r",
+    "--revision",
+    default=None,
+    type=int,
+    help='Ansys Revision number, e.g. "241" or "232". If none is specified\
+, uses the default from ansys-tools-path',
+)
+@click.argument("args", nargs=-1)
+def cli_env(revision: int, args):
+    """CLI tool to load the environment required to run in Linux.
+
+    Parameters
+    ----------
+    revision : int
+        The Ansys Revision number.
+
+    USAGE:
+    The following example demonstrates the main use of this tool:
+        $ mechanical-env -r 232 -- python test.py
+        $ mechanical-env -- make -C doc html
+    """
+    #  Should not update env variables in Windows
+    if os.name == "nt":
+        print("This feature is not available in Windows !")
+        return True
+
+    # Gets the revision number
+    if not revision:
+        exe, version = atp.find_mechanical()
+    else:
+        exe, version = atp.find_mechanical(version=revision)
+    version = int(version * 10)
+
+    env = os.environ.copy()
+
```

Comment:
```suggestion
    AWP_ROOT = env.get(f"AWP_ROOT{version}")
    DS_INSTALL_DIR = env.get("DS_INSTALL_DIR")
```

Reply:
replace env.get(...) lines with AWP_ROOT and DS_INSTALL_DIR below to make it a little cleaner